test/cql-pytest/conftest.py

```diff
@@ -36,6 +36,11 @@ def pytest_addoption(parser):
         help='CQL server port to connect to')
     parser.addoption('--ssl', action='store_true',
         help='Connect to CQL via an encrypted TLSv1.2 connection')
+    # Used by the wrapper script only, not by pytest, added here so it appears
+    # in --help output and so that pytest's argparser won't protest against its
+    # presence.
+    parser.addoption('--omit-scylla-output', action='store_true',
+        help='Omit scylla\'s output from the test output')
```

Comment:
An alternative to adding this option to conftest.py could have been for "run" to emove this option from argv after noticing it, and not pass it to pytest. I guess the benefit of your approach is to add it to "--help", but I am not sure how useful this is - but what the heck :-)